azure-pipelines.yml

```diff
@@ -65,7 +65,7 @@ stages:
           pool:
             ${{ if eq(variables['System.TeamProject'], 'public') }}:
               name: NetCore-Public
-              demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre.Open
+              demands: ImageOverride -equals windows.vs2022preview.amd64.open
             ${{ if ne(variables['System.TeamProject'], 'public') }}:
               name: NetCore1ESPool-Internal
               demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Main
```

Comment:
@jonfortescue what's the equivalent of this? My best guess is one of the "scout" variants?

Reply:
That image doesn't exist so I'm not sure that this works tbh. "Main" seems to me to imply that just normal windows.vs2022.amd64 would work; if you want the "scout" variants though that would be fine as well.